proto/system_config.proto

```diff
@@ -114,6 +114,9 @@ message DaqConfig {
   
   // Set time between port disconnect and host tests shutdown
   int32 port_flap_timeout_sec = 48;
+
+  // USI url
+  string usi_url = 49;
```

Comment:
I would encapsulate this in a message -- it has a feeling that eventually it might have more parameters.